src/components/dialog/demoOpenFromCloseTo/script.js

```diff
@@ -0,0 +1,38 @@
+angular.module('dialogDemo2', ['ngMaterial'])
+
+.controller('AppCtrl', function($scope, $mdDialog) {
+  $scope.showFromLeft = function() {
+    $mdDialog.show(
+      $mdDialog.alert()
+        .clickOutsideToClose(true)
+        .title('Opening from the left')
+        .content('Closing to the right!')
+        .ariaLabel('Left to right demo')
+        .ok('Nice!')
+        // You can specify either sting with query selector
+        .openFrom('#left')
+        // or an element
+        .closeTo(angular.element(document.querySelector('#right')))
```

Comment:
we should simply support .closeTo('#right'); where internal we use `querySelector()`

> also perhaps we provide a second argument `.closeTo(  '#right',  '#nonBodyParent' )`